hashes/contrib/test.sh

```diff
@@ -17,6 +17,14 @@ if cargo --version | grep nightly >/dev/null; then
     NIGHTLY=true
 fi
 
+# Pin dependencies as required if we are using MSRV toolchain.
+if cargo --version | grep "1\.48"; then
+    # 1.0.157 uses syn 2.0 which requires edition 2021
+    cargo update -p serde --precise 1.0.156
+    # 1.0.108 uses `matches!` macro so does not work with Rust 1.41.1, bad `syn` no biscuit.
+    cargo update -p syn --precise 1.0.107
```

Comment:
Actually `matches` is not longer a problem since we bumped MSRV.

Reply:
Oh yeah, cheers. I'll remove the other ones too (in a separate PR).